model/LtiAssignmentAuthorizationService.php

```diff
@@ -35,10 +34,10 @@
  * @package oat\ltiDeliveryProvider\model
  * @author Aleh Hutnikau, <hutnikau@1pt.com>
  */
-class LtiAssignment extends GroupAssignment implements AssignmentService
+class LtiAssignmentAuthorizationService extends ConfigurableService
 {
     const LTI_MAX_ATTEMPTS_VARIABLE = 'custom_max_attempts';
-    const LTI_SERVICE_ID = 'ltiDeliveryProvider/assignment';
+    const SERVICE_ID = 'ltiDeliveryProvider/LtiAssignmentAuthorization';
```

Comment:
unfortunately you cannot rename the constant used to extract the service you have to mark it as deprecated and use the new one.  because in the newest version you need to  unregister  the old service-> `assignment` config deleted .

Ex. Start  >2.0.0 to 7.3.0 The new service it's registered but the old config will remain the same
Start 1.7.1< to 7.3.0 The new service it's registered and the old config does not exist.